backend/src/backend/main.py

```diff
@@ -95,20 +98,41 @@ async def add_book(book_payload: BookAddSchema):
                 kind=KIND_TEXT,
             )
 
+    try:
+        title = await Title.objects.get_or_create(
+            ident=get_ident_from_name(book_payload.metadata["Name"])
+        )
+    except ValueError as exc:
+        raise HTTPException(status_code=400, detail=f"Exception: {exc}") from exc
+
+    await book.update(title=title)
+
+    for metadata in await BookMetadata.objects.all(book=book.id):
+        await TitleMetadata.objects.create(
+            title=title.ident,
+            name=metadata.name,
+            bin_value=metadata.bin_value,
+            value=metadata.value,
+            kind=metadata.kind,
+        )
+
     for tag_name in book_payload.metadata["Tags"].split(";"):
         book_tag = await BookTag.objects.get_or_create(name=tag_name)
         await book.tags.add(book_tag)
+        if not tag_name.startswith(
```

Comment:
Not including the `:` is too loose here. Actually, let's match them properly as suggested

```py
if re.match(r"_(sw|ftindex|pictures|videos|details):(yes|no)", tag_name):
```